creator-node/src/routes/nodeSync.js

```diff
@@ -38,13 +40,16 @@ module.exports = function (app) {
         models.Track.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs }, transaction: t }),
         models.File.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs }, transaction: t })
       ])
-      await t.commit()
+      await t.commit() // why do we have this if we're not making changes?
 
       /** Bundle all data into cnodeUser objects to maximize import speed. */
 
       const cnodeUsersDict = {}
-      cnodeUsers.forEach(cnodeUser => {
-        // Convert sequelize object to plain js object to allow adding additional fields.
+      const TTL = config.get('redisRehydrateCacheTTL')
+
+      const cnodeUserUUIDsOfFilesToRehydrateArrayPromises = []
```

Comment:
this is declared a const but being updated later

Reply:
if you're referring to https://github.com/AudiusProject/audius-protocol/pull/825/files#diff-6296aeaad2e77b78d513911233301094R69, it's actually 2 different arrays! 
`cnodeUserUUIDsOfFilesToRehydrateArrayPromises` and `cnodeUserUUIDsOfFilesToRehydrateArray`